particles/Notification/Notification.arcs

```diff
@@ -0,0 +1,6 @@
+
+particle Notification in './source/Notification.js'
```

Comment:
do we need both - this one, and the one particles/Pipes/Notification.arcs?

Reply:
Fair question ... I'll clean this up as a later step if you don't mind. There is actually a third one pending to accommodate the hackathon demo!